.gitignore

```diff
@@ -1,2 +1,2 @@
-fakemachine
+/fakemachine
```

Comment:
can you please split into another PR

Reply:
Done, see https://github.com/go-debos/fakemachine/pull/120